Lib/multiprocessing/context.py

```diff
@@ -256,6 +256,8 @@ def get_start_method(self, allow_none=False):
     def get_all_start_methods(self):
         if sys.platform == 'win32':
             return ['spawn']
+        elif sys.platform == 'darwin':
+            return ['spawn', 'fork', 'forkserver']
```

Comment:
Is HAVE_SEND_HANDLE guaranteed to be defined on macOS?

Reply:
I'm think it's always set from a certain version (could not find documentation for that), the example of the code that defaults to this case and to 'fork' is in the bugs.python.